modules/data_dictionary_widget/src/Fields/FieldCreation.php

```diff
@@ -61,7 +61,6 @@ protected static function createField(string $field, array $field_json_metadata,
         '#type' => 'textfield',
         '#required' => TRUE,
         '#title' => t('Identifier'),
-        '#default_value' => $field_json_metadata['identifier'] ?? '',
```

Comment:
Just curious as to why this was removed, possibly it wasn't necessary?

Reply:
Ya it was duplicate as there is another line in the same array setting the value to the variable, see https://github.com/GetDKAN/dkan/pull/4289/files/beb8bbad16647ffa8fc03efef5b8782d297ae26b#diff-3d012ae0c7bc8097d5d36c410d4df6d57c3c89124b4ddc1ee1bd5015afd08e7aL66